src/Sql/Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -196,10 +197,18 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase<AzureSqlDatabaseCr
         /// <summary>
         /// Gets or sets the number of readonly replicas for the Azure Sql database
         /// </summary>
+        [CmdletParameterBreakingChange("ReadReplicaCount", ReplaceMentCmdletParameterName = "HighAvailabilityReplicaCount")]
         [Parameter(Mandatory = false,
             HelpMessage = "The number of readonly secondary replicas associated with the database to which readonly application intent connections may be routed. This property is only settable for Hyperscale edition databases.")]
         public int ReadReplicaCount { get; set; }
```

Comment:
You can set ReadReplicaCount  as the alias of HighAvailabilityReplicaCount  to avoid current breaking changes and remove the alias ReadReplicaCount in next major release

Reply:
I have changed ReadReplicaCount to be an alias of HighAvailabilityReplicaCount as requested.  Thanks.